config/google_cloud_exporter/prerequisites.md

```diff
@@ -0,0 +1,72 @@
+# Google Cloud Exporter Prerequisites
```

Comment:
Could this be rolled into a README.md, with a short desc of what ties these configs together (e.g. "This directory's subdirectories contain configs that export to Google Cloud Platform"). I think it'd be nicer for people looking through github, since it shows README files when you are looking at a particular directory.

Reply:
I plan on linking to `Example` configs or explaining it more in the base readme once these are in.

Reply:
Honestly I think just rename this file to `README.md` and have this be a section. Allowing us to add more sections if need be

Reply:
Good points, will handle it.